scripts/import/import_all_sv_data.pl

```diff
@@ -1554,11 +1583,13 @@ sub parse_9th_col {
         $phe = replace_char($phe);
 
         $phe =~ s/__/, /g;
-        $info->{phenotype}{$phe} = 1;
+
+        $phenotype_data = 1;
```

Comment:
Can a comment be added for the meaning of `$phenotype_data =1` and value of `$info->{phenotype}{$phe}`  if `$skip_phenotypes =1`? If the phenotype is to be skipped, why is important to have $phenotype_data =1?

Reply:
Let me know if the comments are not clear.